apps/prairielearn/src/pages/instructorAssessmentQuestions/instructorAssessmentQuestions.ejs

```diff
@@ -176,6 +193,31 @@
                   <% }); %>
                   <% } %>
                 </td>
+                <td class="text-right">
+                  <div class="dropdown js-question-actions">
+                    <button type="button" class="btn btn-secondary btn-xs dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                      Action <span class="caret"></span>
+                    </button>
+
+                    <div class="dropdown-menu">
+                      <% if (authz_data.has_course_instance_permission_edit) { %>
+                        <button
+                          id="row<%= iRow %>PopoverBreakButton"
+                          class="dropdown-item js-mark-variant-as-broken"
+                          data-toggle="popover"
+                          data-trigger="click"
+                          data-container="body"
+                          data-html="true"
+                          data-placement="auto"
+                          title="Confirm mark as broken"
+                          data-content="<%= include('popoverBreakVariants', {iRow: iRow, question: question}) %>"
```

Comment:
In PT I've generally moved to use modals for this kind of confirmation, rather than popovers. Mainly because the UI is nicer (no need for all the janky handling of popover show/hide state) and because it feels like a more serious event when you agree to a modal.

I'm not asking for a change here, just commenting.

Reply:
Heh, in the previous comment I said basically the same thing. I'd be happy to switch to a modal here.

Reply:
Changed to modals in 7e0b69c and 66edad4.

Reply:
Great minds... 😂 🙏